spec/controllers/lists_controller_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
-
+require 'support/database_cleaner'
```

Comment:
This database_clener require should not be needed. Ensure the `spec/support` requires line is uncommented in rails_helper.rb